web-components/src/components/inputs/DatePickField.tsx

```diff
@@ -1,60 +1,148 @@
 import React from 'react';
-import { makeStyles, DefaultTheme as Theme } from '@mui/styles';
 import { FieldProps } from 'formik';
-import { DatePicker } from '@mui/lab';
-import TextField from '@mui/material/TextField';
+import { makeStyles, DefaultTheme as Theme } from '@mui/styles';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { SxProps, TextField } from '@mui/material';
+import { TextFieldProps } from 'formik-mui';
 
 import FieldFormControl from './FieldFormControl';
 
 interface DatePickProps extends FieldProps {
   className?: string;
+  sx?: SxProps<Theme>;
   label: string;
   optional?: boolean;
   placeholder?: string;
 }
 
-interface StyleProps {
+interface StyleProps extends TextFieldProps {
+  errors: boolean;
+  optional: boolean;
   disabled?: boolean;
 }
 
 const useStyles = makeStyles<Theme, StyleProps>((theme: Theme) => ({
-  inputWrap: {
-    marginTop: theme.spacing(5),
-  },
-  input: props => ({
-    border: `1px solid ${theme.palette.grey[100]}`,
-    display: 'flex',
-    flexDirection: 'column',
-    borderRadius: '5px',
-    background: theme.palette.primary.main,
-    color: props.disabled
-      ? theme.palette.info.main
-      : theme.palette.primary.contrastText,
-    margin: theme.spacing(3.25, 0, 0),
-    transition: '300ms ease-in-out;',
-    [theme.breakpoints.up('sm')]: {
-      padding: theme.spacing(3.5),
-      fontSize: `${theme.spacing(3.5)}`,
+  root: props => ({
```

Comment:
These are mostly copy-pasted from `TextField` because I can only pass in a component with vanilla MUI TextField props.

I have had a hard time converting our TextField to `styled`. I don't know why, but passing in the props has not been working for me. 

@haveanicedavid can you help?

Reply:
any updates on this? @mhagel

Reply:
@blushi I think I should give up on forcing our TextField component here. I have tried several strategies, and nothing is working well. Maybe we can pair on it, but I'm stuck.

Reply:
could you create a follow-up task to try to clean that up then? and add a TODO in the code referencing this issue?